tests/parity_fastapi/ls-server.py

```diff
@@ -9,6 +9,15 @@
 
 import litserve as ls
 
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+device = "mps" if torch.backends.mps.is_available() else device
+conf = {
+    "cuda": {"batch_size": 8, "workers_per_device": 1},
+    "cpu": {"batch_size": 4, "workers_per_device": 1},
+    "mps": {"batch_size": 4, "workers_per_device": 1},
+}
```

Comment:
```suggestion
DEVICE = "cuda" if torch.cuda.is_available() else "cpu"
DEVICE = "mps" if torch.backends.mps.is_available() else DEVICE
CONF = {
    "cuda": {"batch_size": 8, "workers_per_device": 1},
    "cpu": {"batch_size": 4, "workers_per_device": 1},
    "mps": {"batch_size": 4, "workers_per_device": 1},
}
```
I may consider using capital as a kind of constant, right?